application/tests/Feature/Http/Controllers/InstitutionControllerIndexTest.php

```diff
@@ -13,7 +13,7 @@
 use Tests\Feature\RepresentationHelpers;
 use Tests\TestCase;
 
-class InstitutionControllerTest extends TestCase
+class InstitutionControllerIndexTest extends TestCase
```

Comment:
Changes in this class are not relevant to this PR

Reply:
I would not say that it is _not relevant_ – it creates a consistent naming system for institution-related tests, ensuring future readers understand which file has which responsibility. However, I do concur that this change is _not critical_ – it’s a "nice-to-have".

<img width="358" alt="image" src="https://github.com/keeleinstituut/tv-authorization/assets/26142658/b55c9d8d-b4c8-4eee-8b86-60a122efab69">